build-tools/scripts/RunTests.targets

```diff
@@ -76,6 +76,7 @@
     <MSBuild 
         Projects="$(_TopDir)\tests\RunApkTests.targets"
         Condition=" '$(Configuration)' == 'Release' "
+        Properties="AotAssemblies=True"
```

Comment:
Ok yeah, this does look like I left something out.

The only thing I found with the `MSBuild` task is that if you set `Properties` you lose any existing ones set.

So you may need:
```
Properties="Configuration=$(Configuration);AotAssemblies=True"
```

Looking at it, I _think_ `Configuration` is the only other value you need.

Reply:
I have checked the build log and it runs with Release configuration, so I think this is not needed.

```
11:50:38 		Configuration = Release
```